src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -47,10 +50,35 @@
 #include "ui_automatedrssdownloader.h"
 #include "automatedrssdownloader.h"
 
+class AutomatedRssDownloader::DownloadRuleListMatchState
```

Comment:
No problems - I can rework it on the weekend if needed. Working on things today because it's a public holiday here today (Australia Day).